documentation/api/api-explorer.yml

```diff
@@ -277,7 +277,7 @@ components:
           description: >-
             The total value held in unspent outputs owned by the given address 
             (includes funds held in storage deposit).
-        sigLockedBalance:
+        availableBalance:
```

Comment:
also update the description?

Reply:
I didn't feel that it needed to be updated really

Reply:
well, to me it seems a bit wrong now since `availableBalance` doesn't just consider signature locked outputs anymore, but also Timelocks and Expirations.

Reply:
well that description is for total balance

Reply:
well, yeah, it's just github that shows you the L277-279 because it things that would be helpful while it is just confusing here.. I was actually referring to the description of `availableBalance` in L283-284, which says:
"The sum of value held in unspent outputs owned by the given address that are signature locked ("trivially unlockable")." which is not correct anymore.

Reply:
Ahh. What do you think it should say?